config/initializers/resque.rb

```diff
@@ -1,11 +1,7 @@
 # frozen_string_literal: true
 
 if defined?(Resque)
-  Resque.redis = Redis.new(
-    host: Rails.application.secrets[:redis_host],
-    port: Rails.application.secrets[:redis_port],
-    password: Rails.application.secrets[:redis_pass]
-  )
+  Resque.redis = Redis.new(url: Rails.application.secrets[:redis_url])
```

Comment:
question: Is the motivation here to simplify deployment with Heroku?

Reply:
Yes. Heroku doesn't give us separate connection variables for ports, password, host and so on. It's just a single REDIS_URL, so rather than having config/secrets.yml parse that out into separate bits, I reconfigured the redis setup to take the url.